packages/telemetry/src/span-exporters/cloud-span-exporter.ts

```diff
@@ -51,8 +53,10 @@ export class OTLPTraceExporter extends OTLPTraceExporterHttp {
       return
     }
 
+    // Continue to send this header for passivity until the cloud is released.
```

Comment:
Is there an issue logged to remove this one the cloud is updated?

Reply:
Nope, I was just planning to do it myself once i was notified that cloud had released.